CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## Changelog
 
+### 2018.11.1
+
+**Released**: 2018.11.13
+
+**Commit Delta**: [Change from 2018.10.1 release](https://github.com/plus3it/spel/compare/2018.10.1...2018.11.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2018.11.1/manifests>
+
+**Summary**:
+
+*   Minor Release
```

Comment:
Can you add a bullet before "Minor Release" (same level) saying that the Python version has been upgraded from python34 to python36?

Reply:
updated.  Also added a bullet pertaining to builders for EL6 has been deprecated.